rest-new/src/main/java/org/jboss/pnc/rest/endpoints/ProductMilestoneEndpointImpl.java

```diff
@@ -96,4 +102,16 @@ public void closeMilestone(String id, ProductMilestone productMilestone) {
     public void cancelMilestoneClose(String id) {
         productMilestoneProvider.cancelMilestoneCloseProcess(id);
     }
+
+    @Override
+    public ValidationResponse validateVersion(VersionValidationRequest versionRequest) {
+        return ValidationResponse.builder()
+                .isValid(Pattern.matches(Patterns.PRODUCT_MILESTONE_VERSION, versionRequest.getVersion()))
+                .hints(
+                        Lists.newArrayList(
+                                "Product Milestone version consists of 2 or 3 numeric components followed by a string"
```

Comment:
I would try to make it a little bit shorter, something like:
`Allowed format consists of 2 or 3 numeric components (separated by a dot) followed by a string qualifier starting with a character, eg. 3.0.0.GA, 1.0.11.CR2.ER1, 3.0.CR2`

Reply:
Will change!